docs/docs/get_started.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Get Started
 id: get_started
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/get_started.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Overview/overview.htm
```

Comment:
let's change this to link to the quick start: https://docs.secretless.io/Latest/en/Content/Get%20Started/quick_start.htm